controllers/tab/user/ProfileTabHandler.inc.php

```diff
@@ -300,15 +301,25 @@ public function changePassword($args, $request)
     public function savePassword($args, $request)
     {
         $this->setupTemplate($request);
-        $passwordForm = new ChangePasswordForm($request->getUser(), $request->getSite());
+
+		$user = $request->getUser();
+
+        $passwordForm = new ChangePasswordForm($user, $request->getSite());
         $passwordForm->readInputData();
 
         if ($passwordForm->validate()) {
+
             $passwordForm->execute();
+
+			$sessionManager = SessionManager::getManager();
+            $sessionManager->invalidateSessions($user->getId(), $sessionManager->getUserSession()->getId());
+
             $notificationMgr = new NotificationManager();
-            $notificationMgr->createTrivialNotification($request->getUser()->getId());
-            return new JSONMessage(true);
+            $notificationMgr->createTrivialNotification($user->getId());
+            
+			return new JSONMessage(true);
```

Comment:
There are still some tab-based indents here; there should be pre-commit git hooks that run the code formatter over the code (using http://captainhook.info). Could you check into whether that's working?

Reply:
fixed it for this file and other files . should be fine now .